### PR TITLE
all: continue scanning when error, no empty labels

### PR DESCRIPTION
**Pull request by manelmontilla**
This PR introduces the following modifications:

- Make sure all the created vertices have a non empty label.
- Modifies the process of scanning accounts so, now, it will scan all the accounts even if an uncontrolled error is raised when scanning any of them.
- It improves the ``test scan`` by adding an EC2 instance and an AMI with tags.

### CODE REVIEW

File: graph_altimeter/scan/neptune_client.py [hunk 1]
```diff
@@ -148,3 +151,13 @@ def __write_edges(self, g, edges, scan_id):
                     raise NeptuneLoadGraphException(
                         f"Error loading edge {r} " f"with {str(t.bytecode)}"
                     ) from err
+
+    def __label_for(self, node_id):
+        """
+        Builds the label of a vertex or edge from his id.
```
Comment (jroimartin) [inline graph_altimeter/scan/neptune_client.py]: s/his/its/ ?

File: tests/conftest.py [hunk 2]
```diff
@@ -468,3 +483,42 @@ def create_bucket(name, account_id, region_name):
         f"arn:aws:s3:{region_name}:{account_id}:bucket/{name}",
         creation_date,
     )
+
+
+def create_instance(image_id, instance_type, tags, region_name):
+    """Creates and instance using the given `image_id`, for instance:
+    'ami-one', `instance_type`, for instance: 't2.micro', `tags`, for instance
+    [{'Key': 'tag_one', 'value': 'value'}] and `region_name`, for instance:
```
Comment (jroimartin) [inline tests/conftest.py]: `[{'Key': 'tag_one', 'value': 'value'}]` seems a bit weird. Specially the `'value': 'value'` part. What about `[{'Key': 'tag_one', 'value': 'value_one'}]`?

File: tests/conftest.py [hunk 3]
```diff
@@ -468,3 +483,42 @@ def create_bucket(name, account_id, region_name):
...
+                "ResourceType": "instance",
+                "Tags": tags,
+            }
+        ],
+    )
+    return instance["Instances"][0]["InstanceId"]
+
+
+def create_image(instance_id, name, region_name):
+    """Creates an image from the given ec2 ``instance_id``, the given ``name``
```
Comment (jroimartin) [inline tests/conftest.py]: s/, the/and the/ ?

File: tests/conftest.py [hunk 4]
```diff
@@ -468,3 +483,42 @@ def create_bucket(name, account_id, region_name):
...
+    client = boto3.client("ec2", region_name=region_name)
+    image_id = client.create_image(
+        Name=name,
+        InstanceId=instance_id
+    )["ImageId"]
+    return image_id
+
+
+def tag_resource(arn, tags):
+    """Tags the resource identified by specified ``arn`` with the tags
```
Comment (jroimartin) [inline tests/conftest.py]: s/by/by the/ ?

**Review by jroimartin — CHANGES_REQUESTED** [review]

File: tests/conftest.py (hunk 2 above)
Comment (manelmontilla) [reply tests/conftest.py]: 👍

File: graph_altimeter/scan/neptune_client.py (hunk 1 above)
Comment (manelmontilla) [reply graph_altimeter/scan/neptune_client.py]: 👍

File: tests/conftest.py (hunk 3 above)
Comment (manelmontilla) [reply tests/conftest.py]: 👍

File: tests/conftest.py (hunk 4 above)
Comment (manelmontilla) [reply tests/conftest.py]: 👍

**Review by jroimartin — APPROVED** [review]
LGTM

**Pull request merged**